sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_model_configurations.py

```diff
@@ -6,17 +6,24 @@
 
 
 class AzureOpenAIModelConfigurationBase(TypedDict):
```

Comment:
just curious, why need AzureOpenAIModelConfigurationBase and OpenAIModelConfigurationBase?